api/src/main/java/org/pmiops/workbench/access/AccessModuleServiceImpl.java

```diff
@@ -108,17 +108,19 @@ public void updateBypassTime(
   }
 
   @Override
-  public void updateCompletionTime(
+  public DbUserAccessModule updateCompletionTime(
```

Comment:
I think "return the updated object" is reasonable behavior for methods that create or modify data. The only complication here is the IDENTITY logic, but we expect to eventually remove that logic.